basic-algorithms.js

```diff
@@ -1,7 +1,41 @@
-// Names and Input
+//variables 
+var hacker1 = "Samuel"
+var hacker2 = window.prompt("what is the navigators name?")
 
 
-//Conditionals
+//console logs
+console.log(`The drivers name is ${hacker1}`);
+console.log(`The navigators name is ${hacker2)}`)
 
 
-// Lorem ipsum generator
+//Let's get looping
+for (var i = 0; i < hacker1.length || hacker2.length; i++){
```

Comment:
The challenge here was to compare the length of two element. It would have been easier to just compare **element1.length** and **element2.length** with no need to use a for loop since you can get the length of a string directly with the **".length()"** property.  Then you only return the one that is longer (**if element1.length < element2.length return element1)**